src/RequestPage.php

```diff
@@ -215,7 +215,7 @@ function requestActionsForm(AccountRequest &$accountRequest, string $userContext
 		);
 		$disp .= Html::closeElement('form');
 		
-		$output->addHTML($disp);
+		$output->addHTML(Linker::formatComment($disp));
```

Comment:
This is not the body of the comment, this is the entire body of the page. Also, this should only apply to admin comments.